phpcs.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0"?>
```

Comment:
I've made the change as I don't necessarily disagree with your suggestion.

> Because we're distributing it, but others may want to override this locally with their own PHPCS

As it's a library, people shouldn't adjust the code - even for style - when using it.

If it's being used in a larger project, PHPCS will use the `phpcs.xml(.distr)` file in the root - or directories above this - of the larger project anyway.
It will not respect `phpcs.xml` files in directories below the root of the project, so having a `phpcs.xml` in a subdirectory (`/tgmpa`) will not affect the CS run.

> - maybe to test it against the next version of WPCS or some other reason.

The XML file is independent of version - apart from new properties which may be introduced -. What version is used to run it against depends on the version set up locally or in the test environment.